src/comment-utils.js

```diff
@@ -0,0 +1,209 @@
+'use strict'
```

Comment:
This module is just copying some core comment-printing utils from Prettier. This is similar to what eg the Prettier PHP plugin does, I remember seeing a Github issue about exposing a plugin API for comment helpers